galaxy/Dockerfile

```diff
@@ -65,8 +68,6 @@ RUN mkdir -p $GALAXY_CONFIG_DIR $GALAXY_CONFIG_DIR/web && chown -R $GALAXY_USER:
 RUN su $GALAXY_USER -c "cp $GALAXY_ROOT/config/galaxy.ini.sample $GALAXY_CONFIG_FILE"
 ADD ./reports_wsgi.ini.sample $GALAXY_CONFIG_DIR/reports_wsgi.ini
 ADD ./htpasswd /etc/nginx/htpasswd
-ADD ./servers.ini $GALAXY_CONFIG_DIR/servers.ini
-RUN cat $GALAXY_CONFIG_DIR/servers.ini >> $GALAXY_CONFIG_FILE && rm $GALAXY_CONFIG_DIR/servers.ini
```

Comment:
I still don't know how Galaxy handles the handler ports. Because these are now not specified explicitly. But I can run tools so I think it's working and all handlers are used randomly.

What I still need to figure out is how to dynamically adopt `<plugins workers="4">` in `job_conf.xml`. But this is not strictly related to this PR.